src/pathways/services/pathwayUtils.service.ts

```diff
@@ -0,0 +1,151 @@
+import * as _ from 'lodash';
+import * as fp from 'lodash/fp';
+import * as jsnx from 'jsnetworkx';
+
+export class PathwayUtilsService {
+    // Move to Escher once it's functional
+    DEFAULT_COFACTORS = new Set([
+        'MNXM3',  // 'ATP'
+        'MNXM7',  // 'ADP'
+        'MNXM8',  // 'NAD(+)'
+        'MNXM10',  // 'NADH(2-)'
+        'MNXM5',  // 'NADP(+)
+        'MNXM6',  // 'NADPH'
+        'MNXM51',  // 'GTP'
+        'MNXM30',  // 'GDP'
+        'MNXM12',  // 'CoA'
+        'MNXM80',  // 'UMP(2-)'
+        'MNXM1',  // 'H(+)'
+        'MNXM4',  // 'O2'
+        'MNXM13',  // 'CO(2)'
+        'MNXM2',  // 'H2O'
+    ]);
+
+    // Utility function since the Set interface is rather poor at the moment
+    public setUnion(...sets) {
+        return sets.reduce((ret, nextSet) => {
+            nextSet.forEach((e) => {
+                ret.add(e);
+            });
+            return ret;
+        }, new Set());
+    }
+    /**
+     * Extracts the metabolite names from a list of reactions
+     * The returned list is sorted and unique.
+     * @param reactions 
+     */
+
+    public reactionsMetabolites(reactions) {
```

Comment:
Either a static method or a function outside of the class is appropriate. It's misleading to have it on the object when it does not use the object. 

A helpful name is more valuable than documenting a function, even if you need to use a few words to describe it.